examples/pxScene2d/src/pxVideo.cpp

```diff
@@ -486,19 +553,36 @@ rtError pxVideo::stop()
 	return RT_OK;
 }
 
-rtError pxVideo::setSpeed(float speed, float overshootCorrection)
+rtError pxVideo::setSpeed(int speed, int overshootCorrection)
 {
 	if(mAamp)
 	{
-		mAamp->SetRate(speed);
+		mAamp->SetRate(speed, overshootCorrection);
```

Comment:
This function existed, however it did not use overshootCorrection parameter. I passed it to PlayerInstanceAAMP::setRate.